client/asset/btc/btc.go

```diff
@@ -3985,7 +4100,9 @@ func (btc *baseWallet) AuditContract(coinID, contract, txData dex.Bytes, rebroad
 	// Broadcast the transaction, but do not block because this is not required
 	// and does not affect the audit result.
 	if rebroadcast && tx != nil {
+		btc.subprocessWg.Add(1)
 		go func() {
+			defer btc.subprocessWg.Done()
 			if hashSent, err := btc.node.sendRawTransaction(tx); err != nil {
 				btc.log.Debugf("Rebroadcasting counterparty contract %v (THIS MAY BE NORMAL): %v", txHash, err)
```

Comment:
This uses the `subprocesWg` for synchronization, but the call to `sendRawTransaction` ultimately uses the `baseCtx`, not the passed `ctx`.  I don't see how that might cause a deadlock or anything, but it's fundamentally incorrect to do that with this 2-context pattern, and another reason that I think this pattern is probably not the right solution.